code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm

```diff
@@ -104,30 +104,15 @@
 	if(recovery_time >= world.time)
 		return
 	. = ..()
-	if(. && isliving(target))
-		var/mob/living/L = target
-		if(L.stat != DEAD)
-			if(!client && ranged && ranged_cooldown <= world.time)
-				OpenFire()
-
-			if(L.health <= HEALTH_THRESHOLD_DEAD && HAS_TRAIT(L, TRAIT_NODEATH)) //Nope, it still gibs yall
-				devour(L)
-		else
-			devour(L)
-
-/mob/living/simple_animal/hostile/megafauna/proc/devour(mob/living/L)
-	if(!L)
-		return FALSE
-	visible_message(
-		"<span class='danger'>[src] devours [L]!</span>",
-		"<span class='userdanger'>You feast on [L], restoring your health!</span>")
-	if(!is_station_level(z) || client) //NPC monsters won't heal while on station
-		adjustBruteLoss(-L.maxHealth/2)
-	for(var/obj/item/W in L)
-		if(!L.dropItemToGround(W))
-			qdel(W)
-	L.gib()
-	return TRUE
```

Comment:
I don't think this proc should go away entirely - megafauna should still have some means of healing after scoring a kill, though I'm not sure by exactly what means. 

Do you have any thoughts on it?

Reply:
I believe that with enough dedication and perseverance dying/being recovered/being revived you probably deserve the kill. I don't think there will be any non-snowflake way of making them heal after each kill, and just the time investment in beginning the fight, dying, waiting to be recovered, the paramedic dodging the megafauna, and being brought back to the station (?) and being revived, not by defib mind you because the organs are probably going to be rotten, that you won't get far just trying to brute force bubblegum or something.